code/game/objects/items/reagent_containers/autoinjectors.dm

```diff
@@ -143,3 +144,13 @@
 		..()
 		reagents.add_reagent("inaprovaline", 15)
 		update_icon()
+
+/obj/item/reagent_container/hypospray/autoinjector/hypervene
+	name = "\improper Hypervene autoinjector"
+	desc = "An auto-injector loaded with 3 units of Hypervene, an emergency medicine that rapidly purges chems. Causes pain and vomitting."
+	icon_state = "anesthetic" //TEMP
+
+	/obj/item/reagent_container/hypospray/autoinjector/hypervene/New()
```

Comment:
Indentation. All of these `New()` proc redefinitions should be at the same level as the defined objects, not inside them.